docs/user-guide/release-notes/2.15.x.rst

```diff
@@ -2,14 +2,38 @@
 Pulp 2.15 Release Notes
 =======================
 
+Pulp 2.15.3
+===========
+
+Bug Fixes
+---------
+
+See the list of :fixedbugs_pulp:`2.15.3`
+
+Pulp 2.15.2
+===========
+
+Bug Fixes
+---------
+
+See the list of :fixedbugs_pulp:`2.15.2`
+
+Pulp 2.15.1
+===========
+
+Bug Fixes
+---------
+
+See the list of :fixedbugs_pulp:`2.15.1`
+
 Pulp 2.15.0
 ===========
 
 New Features
 ------------
 
 * Logging of `force_full` being enabled
-  `force_full` is usefull for debugging, but it has known performance issues long term.  
+  `force_full` is usefull for debugging, but it has known performance issues long term.
```

Comment:
This is a good change, but probably shouldn't be part of this PR.

Reply:
The point of these PRs are to get all the changes made ready for release, and this is usually the only time the changes go in.

The changes in 2.15.x.rst are cherry-picked commits, fixing an accidental oversight where these commits didn't land in master, and therefore didn't get added to 2.16-release branch.

With that, if you still rather them be a separate pr entirely I can accomodate.

Reply:
As long as this change is merged to master I'm fine with it. I thought this might have happened in the editor solving a merge conflict.